integrationTests/vm/staking/stakingV4_test.go

```diff
@@ -86,6 +86,19 @@ func remove(slice [][]byte, elem []byte) [][]byte {
 	return ret
 }
 
+func getIntersection(slice1, slice2 [][]byte) [][]byte {
+	ret := make([][]byte, 0)
+	for _, value := range slice2 {
+		if searchInSlice(slice1, value) {
+			copiedVal := make([]byte, len(value))
```

Comment:
Better to use copies instead of copied pointers that can change through the testing code. In other tests I modify the initial slices and it messes up tests.